tests/System.Numerics.Tensors.Tests/Tensor.Operations.tt

```diff
@@ -0,0 +1,240 @@
+﻿<#@ template debug="false" hostspecific="false" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ output extension=".cs" #>
+<#@ include file="TensorTemplate.ttinclude" #>// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Numerics.Tensors
+{
+    public static partial class TensorOperations
+    {
+        internal static void ValidateBinaryArgs<T>(Tensor<T> left, Tensor<T> right)
```

Comment:
Why are these methods in the `.tt` file? This class is `partial`, so can they just be in a separate .cs file, to not clutter up the `.tt`?

Reply:
I couldn't think of a good name and it seemed reasonable to just put them in the same file as they were used.